src/symbols.js

```diff
@@ -21,12 +21,16 @@ module.exports = {
     text: {},
 };
 
-function defineSymbol(mode, font, group, replace, name) {
+function defineSymbol(mode, font, group, replace, name, acceptUnicodeChar) {
     module.exports[mode][name] = {
         font: font,
         group: group,
         replace: replace,
     };
+
+    if (acceptUnicodeChar) {
+        module.exports[mode][replace] = module.exports[mode][name];
+    }
```

Comment:
Love this new approach!  Avoids repetition of the Unicode symbol, and makes the decision of "include this unicode character" clear.  I think if we ever want a unicode symbol to point to one not matching the font, then we can manually define that symbol.